service/iot-service-samples/pnp-service-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/PnPNoComponentSample.java

```diff
@@ -0,0 +1,60 @@
+package samples.com.microsoft.azure.sdk.iot;
+
```

Comment:
Can we add a readme explaining the usage of both Device and Service samples togther? And a brief explanation of both the samples?

Reply:
That is a TODO for sure. We have not discussed where to add all that info but it is something we need to do.